test/test_fcl_capsule_capsule.cpp

```diff
@@ -126,6 +125,7 @@ BOOST_AUTO_TEST_CASE(distance_capsulecapsule_transformZ)
 
 BOOST_AUTO_TEST_CASE(distance_capsulecapsule_transformZ2)
 {
+  const FCL_REAL Pi = std::acos(FCL_REAL(-1.0));
```

Comment:
I would prefer either using boost::math::constants::pi<double> or defining M_PI if not defined.
{{{
#ifndef M_PI
#define M_PI 3.1415926535....
#endif
}}}